cypress/e2e/pages/mostReadPage/index.cy.js

```diff
@@ -63,7 +77,74 @@ const liteTestSuites = testSuites.map(testSuite => {
   };
 });
 
+const atiAnalyticsTestSuites = [
+  {
+    path: '/gahuza/popular/read',
+    runforEnv: ['local', 'test', 'live'],
+    service: 'gahuza',
+    pageIdentifier: 'gahuza.popular.read.page',
+    siteId: 40,
+    applicationType: 'responsive',
+    contentType: 'list-datadriven',
+    useReverb: true,
+    tests: [
+      assertPageView,
+      assertMostReadComponentView,
+      assertMostReadComponentClick,
+      assertDropdownNavigationComponentClick,
+      assertDropdownNavigationComponentView,
+      assertScrollableNavigationComponentView,
+      assertScrollableNavigationComponentClick,
```

Comment:
Assertions in order of how they are displayed on the page
```suggestion
      assertDropdownNavigationComponentClick,
      assertDropdownNavigationComponentView,
      assertScrollableNavigationComponentView,
      assertScrollableNavigationComponentClick,
      assertMostReadComponentView,
      assertMostReadComponentClick,
```